x-pack/plugins/lens/public/trigger_actions/open_lens_config/in_app_embeddable_edit/in_app_embeddable_edit_action_helpers.tsx

```diff
@@ -104,52 +105,55 @@ export async function executeEditEmbeddableAction({
     };
     onUpdate(newAttributes);
   };
+  core.theme.theme$.subscribe(async (theme) => {
+    const Component = await getEditLensConfiguration(core, deps, visualizationMap, datasourceMap);
+    const ConfigPanel = (
+      <EuiThemeProvider colorMode={theme.darkMode ? 'dark' : 'light'}>
```

Comment:
This is the change. 

If there is a different theming API that is better suited for this use case, please lmk.

Reply:
Maybe is better to add this here https://github.com/elastic/kibana/blob/main/x-pack/plugins/lens/public/app_plugin/shared/edit_on_the_fly/get_edit_lens_configuration.tsx#L226 instead of the action.

This dark mode inconsistency doesnt happen to dashboard or Discover. Are you wrapping the Obs AI assistant with the ThemeProvider? 🤔